lib/utils/testLauncherHelper.js

```diff
@@ -395,7 +395,7 @@ function getChildOptions(deviceId, deviceAccessToken, appConfigId, isDebugMode,
 		[envVars.SUITEST_CONFIG_CONTINUE_ON_FATAL_ERROR]: config.continueOnFatalError,
 		[envVars.SUITEST_CONFIG_TIMESTAMP]: config.timestamp,
 		[envVars.SUITEST_LAUNCHER_VERSION]: version,
-		[envVars.SUITEST_USER_CONFIG]: config.config,
+		[envVars.SUITEST_USER_CONFIG]: config.configFile,
 		FORCE_COLOR: true,
```

Comment:
my bad :)

Reply:
maybe it worth to add some ut, which checks that correct env vars is set?